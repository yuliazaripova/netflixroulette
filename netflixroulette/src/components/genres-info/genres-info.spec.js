import * as React from 'react';

import GenresInfo from './genres-info';

describe("GenresInfo component", () => {
  it("should render GenresInfo component", () => {
    const component = shallow(<GenresInfo genre={"testGenre"}/>);
    expect(component).toMatchSnapshot();
  })
})