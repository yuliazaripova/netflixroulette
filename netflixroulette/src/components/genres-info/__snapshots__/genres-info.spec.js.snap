// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenresInfo component should render GenresInfo component 1`] = `
<Fragment>
  <span
    className="genre-info"
  >
    Films by 
    testGenre
     genre
  </span>
</Fragment>
`;
